Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -474,26 +489,24 @@ class BrowserViewController: UIViewController {
         }
     }
     
-    // Because crashedLastSession is sticky, it does not get reset, we need to remember its
-    // value so that we do not keep asking the user to restore their tabs.
-    var displayedRestoreTabsAlert = false
-    
-    var crashedLastSession = false
-
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        if !displayedRestoreTabsAlert && crashedLastSession {
-            displayedRestoreTabsAlert = true
-            showRestoreTabsAlert()
-        } else {
-            tabManager.restoreTabs()
-        }
+        self.checkCrashRestoration()
```

Comment:
`self` not needed. Shouldn't swiftlint catch it btw?

Reply:
There's a [SwiftLint rule](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/Style/ExplicitSelfRule.swift) to catch it when you're not using it (i.e. `checkCrashRestoration`  -> `self.checkCrashRestoration`) but not the other way around?

Reply:
Didn't get a warning on my end...?

Reply:
Na I meant there's no rule for making sure you _don't_ use self explicitly.